Modules/vc-module-pricing/VirtoCommerce.PricingModule.Data/Authorization/PricingAuthorizationHandler.cs

```diff
@@ -0,0 +1,16 @@
+using VirtoCommerce.Platform.Security.Authorization;
```

Comment:
We do not need to create Specific requirement and handler for module permission, we can get policy by name PermissionAuthorizationPolicyProvider, as Authorizeattribute does for permissions